clients/java/src/main/java/io/camunda/client/impl/command/ClockPinCommandImpl.java

```diff
@@ -65,7 +66,12 @@ public FinalCommandStep<PinClockResponse> requestTimeout(final Duration requestT
   @Override
   public CamundaFuture<PinClockResponse> send() {
     final HttpCamundaFuture<PinClockResponse> result = new HttpCamundaFuture<>();
-    httpClient.put("/clock", jsonMapper.toJson(request), httpRequestConfig.build(), result);
+    httpClient.put(
+        "/clock",
+        jsonMapper.toJson(request),
+        httpRequestConfig.build(),
+        PinClockResponseImpl::new,
```

Comment:
💭 Here we have a bit of a different naming schema than the other commands, shouldn't it be `ClockPinResponse` or `PinClockCommandImpl` - I guess the latter one 😄, but that was already there, thus probably irrelevant for this PR.

Reply:
OK, here you go: https://github.com/camunda/camunda/pull/36748/commits/70eb9cfe9606f97aff382ec0e6977e171d7f2dfa

Reply:
Follow-up docs PR: https://github.com/camunda/camunda-docs/pull/6487